design/baremetal-operator/firmware-interface.md

```diff
@@ -0,0 +1,222 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# Support Firmware Interface
```

Comment:
"Update Firmware of BMHs" or something similar?

Reply:
Ack